storyscript/ErrorCodes.py

```diff
@@ -292,6 +292,10 @@ class ErrorCodes:
         'E0154',
         'Type cast to `object` is not allowed.'
     )
+    object_expect_action = (
+        'E0155',
+        'ServiceAction object expected but existing variable `{var}` found.'
```

Comment:
This is too implementation-specific.

```suggestion
        'Service object expected but existing variable `{var}` found.'
```